tests/Mbstring/MbstringTest.php

```diff
@@ -195,7 +195,7 @@ public function testStrCase()
         }
         if (PHP_VERSION_ID >= 70000) {
             // Native iconv() is buggy before PHP 7
-            $this->assertSame('ab', str_replace('?', '', mb_strtolower(urldecode('a%A1%C0b'))));
+            $this->assertSame('ab', str_replace('?', '', mb_strtolower(urldecode('a%A1%C0b'), 'ASCII')));
```

Comment:
why is this included in this PR ? It looks totally unrelated

Reply:
@nicolas-grekas done: https://bugs.php.net/bug.php?id=76319. I've tested that on my local master build and it failed too.

Reply:
@stof I've removed this commit from PR.